Sources/Tracking/Store/SdkConfig.swift

```diff
@@ -18,6 +18,12 @@ public struct SdkConfig {
      */
     public var autoTrackPushEvents: Bool = true
 
+    /**
+     Automatic tracking of screen views will generate `screen`-type events on every screen transition within
+        your application. If you would prefer to handle these yourself, disable this feature
+     */
+    public var autoTrackScreenViews: Bool = true
```

Comment:
Should we make this an opt-in feature?

Reply:
probably yeah. I was writing it with the context of the JS snippet in my head, where we do auto-track
> To capture page view events, add our JavaScript snippet to your site. We will track page views on any page where that snippet is present. The event name of a page view event is the URL of the page that a person visited.

but I think the usage pattern is likely not analogous